ballerina/commons.bal

```diff
@@ -113,15 +113,19 @@ public type ListenerConfiguration record {|
 
 # Record representing a WebSub subscription change request-body.
 #
-# + topic - The topic for which the subscription/unsubscription request is sent
-# + callback - The callback which should be registered/unregistered for the subscription/unsubscription request sent
-# + leaseSeconds - The lease period for which the subscription is expected to be active
-# + secret - The secret to be used for authenticated content distribution with this subscription
+# + topic - The topic for which the subscription/unsubscription request is sent  
+# + callback - The callback which should be registered/unregistered for the subscription/unsubscription request sent  
+# + leaseSeconds - The lease period for which the subscription is expected to be active  
+# + secret - The secret to be used for authenticated content distribution with this subscription  
+# + additionalParams - Additional parameters which needs to be sent with the subscription request
+# + additionalHeaders - Additional HTTP headers which needs to be sent with the subscription request
 public type SubscriptionChangeRequest record {|
     string topic = "";
     string callback = "";
     int leaseSeconds = 0;
     string secret = "";
+    map<string> additionalParams?;
+    map<string> additionalHeaders?;
```

Comment:
Lets use the prefix custom

Reply:
Fixed with [a30a78cbb6c24bd436a9b87f08815fb52f46e6c7](https://github.com/ballerina-platform/module-ballerina-websub/pull/629/commits/a30a78cbb6c24bd436a9b87f08815fb52f46e6c7)